### DISCUSSION TITLE
Sending command by CP (Ex. CMD_LED).

### DISCUSSION THREAD

**Discussion by MiguelASousa** [goToMain/libosdp#179] [Q&A] [answered]
When sending the CMD_LED command, how does the CP know it was successful?
thanks

**Comment by sidcha** [comment]
There is no way to get this information now. I will give some thought to it and add this feature.

**Reply by sidcha** [reply, marked as answer]
@MiguelASousa, I created #182 to track this feature request.

I have a [commit](https://github.com/goToMain/libosdp/commit/0d1cc7759d2ada102fc2900faf3069b735584f10) for you to try and let me know if it suits your need. You can checkout branch [origin/event_notification](https://github.com/goToMain/libosdp/tree/event_notification) locally.

This change has been merged to master.

**Comment by MiguelASousa** [comment]
Good morning, Siddharth Chandrasekaran,

First of all, thank you for your attention.
I was testing the version you created, and it worked well, and is adequate so you can know if the command was successful.

Best regards,
Miguel Sousa

De: Siddharth Chandrasekaran ***@***.***>
Enviada: 4 de junho de 2024 21:21
Para: goToMain/libosdp ***@***.***>
Cc: Miguel Sousa ***@***.***>; Mention ***@***.***>
Assunto: Re: [goToMain/libosdp] Sending command by CP (Ex. CMD_LED). (Discussion #179)


@MiguelASousa<https://github.com/MiguelASousa>, I created #182<https://github.com/goToMain/libosdp/issues/182> to track this feature request.

I have a commit<https://github.com/goToMain/libosdp/commit/0d1cc7759d2ada102fc2900faf3069b735584f10> for you to try and let me know if it suits your need. You can checkout branch origin/event_notification<https://github.com/goToMain/libosdp/tree/event_notification> locally.
